lib/job-iteration/iteration.rb

```diff
@@ -39,6 +39,32 @@ def inspected_cursor
       define_callbacks :start
       define_callbacks :shutdown
       define_callbacks :complete
+
+      class_attribute(
+        :max_job_runtime,
+        instance_writer: false,
+        instance_predicate: false,
+        default: JobIteration.max_job_runtime,
+      )
+
+      singleton_class.prepend(PrependedClassMethods)
+    end
+
+    module PrependedClassMethods
+      def max_job_runtime=(new)
+        existing = max_job_runtime
+
+        if existing && (!new || new > existing)
```

Comment:
Is it worth adding one more test case for the scenario where a global runtime is set, and a class attempts to set `max_job_runtime` to nil, in order to exercise the `if existing && !new` piece of the condition?